tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "skipLibCheck": false,
     "noEmit": true,
-    "noUnusedLocals": false
+    "noUnusedLocals": false,
+    "moduleResolution": "node16"
```

Comment:
I don’t know if you want this here or in your shared tsconfig, but it’d be best to use it to make sure that it keeps working.

Reply:
My latest config has included this so I will remove this line once I upgrade my config.